tests/testthat/test-net-long.R

```diff
@@ -3,7 +3,7 @@ context("Network extended models")
 ################################################################################
 
 test_that("edges models", {
-  skip_on_cran()
+  #skip_on_cran()
```

Comment:
Put this back to uncommented. To run these blocks locally, you can use:
`Sys.setenv("NOT_CRAN"="true")`